### PR TITLE
Add documentation for my

### PR DESCRIPTION
**Pull request by bramkragten**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46058
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2242
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by balloob — APPROVED** [review]

File: source/_integrations/my.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: My Home Assistant
+description: Support for the my.home-assistant.io redirect service.
+ha_category:
+  - Other
+ha_release: 2021.3.0
```
Comment (frenck) [inline source/_integrations/my.markdown:6]: This is incorrect, should have been `2021.3`

File: source/_integrations/my.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ha_domain: my
+ha_iot_class: Calculated
```
Comment (frenck) [inline source/_integrations/my.markdown:8]: The IoT class is not calculated, as this integration does not provide entities and does not dirrive states from other entities. In this case, it should be have nulled.

File: source/_integrations/my.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+This integration handles redirects from the [My Home Assistant](https://my.home-assistant.io) service.
+
+My Home Assistant allows the documentation to link you to specific pages in your Home Assistant instance. See the [My Home Assistant FAQ](https://my.home-assistant.io/faq.html) for more information.
+
+## Configuration
+
+This integration is by default enabled, unless you've disabled or removed the [`default_config:`](https://www.home-assistant.io/integrations/default_config/) line from your configuration. If that is the case, the following example shows you how to enable this integration manually:
```
Comment (frenck) [inline source/_integrations/my.markdown:17]: Don't use full links.

Comment (bramkragten) [reply source/_integrations/my.markdown:17]: https://github.com/home-assistant/home-assistant.io/pull/16458

File: source/_integrations/my.markdown (hunk 2 above)
Comment (bramkragten) [reply source/_integrations/my.markdown:8]: https://github.com/home-assistant/home-assistant.io/pull/16458

File: source/_integrations/my.markdown (hunk 1 above)
Comment (bramkragten) [reply source/_integrations/my.markdown:6]: https://github.com/home-assistant/home-assistant.io/pull/16458

**Pull request merged**